release/1-18/1-18.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: distro.eks.amazonaws.com/v1alpha1
```

Comment:
Let's add the Apache License configuration to these files as well

Reply:
While # is a valid comment in yaml, I'm not a huge fan of uploading these artifacts with comments in them, nor am I a big fan of some logic to strip the comments off.

Reply:
Why do we need to strip comments? Also it's a requirement to add the Apache License to our source code I thought.

Reply:
It just seems really odd to have that copyright comment in the middle of an artifact. We can just skip automating this and someone can copy these files manually with no copyright.

Reply:
I changed this to grep out the comments before uploading them.